### ISSUE TITLE
no forkID received

### ISSUE THREAD

**Issue by gr8h** [0xPolygon/cdk-validium-node#47]
After running `make build-docker` containers `zkevm-sync`, `zkevm-sequencer`, .... are logging `error getting forkIDs. Error: error: no forkID received. It should receive at least one, please check the configuration...`. Is there a miss configs? I used the default file `test.node.config.toml`.

**Comment by HudsonGraeme** [comment]
I also run into this error both on WSL and real Ubuntu 22.04

**Comment by Jasun0925** [comment]
I also encountered this issue on AWS Linux
<img width="1281" alt="image" src="https://github.com/0xPolygon/cdk-validium-node/assets/20411806/f85be2b7-6b72-4428-906d-5b714b024122">

**Comment by Jasun0925** [comment]
@gr8h @HudsonGraeme Have you solved this problem? I also encountered it

**Comment by HudsonGraeme** [comment]
> @gr8h @HudsonGraeme Have you solved this problem? I also encountered it

Yes, I was able to resolve it by downgrading to [`cdk-validium-node:v0.0.2`](https://github.com/0xPolygon/cdk-validium-node/releases/tag/v0.0.2)

**Comment by gr8h** [comment]
Same here

**Comment by UrsaMajor-t** [comment]
Hello, under [cdk-validium-node:v0.0.2](https://github.com/0xPolygon/cdk-validium-node/releases/tag/v0.0.2), what version of zkevm-prover do you use? Can proof be generated normally？ @HudsonGraeme  @gr8h 

> > @gr8h @HudsonGraeme Have you solved this problem? I also encountered it
> 
> Yes, I was able to resolve it by downgrading to [`cdk-validium-node:v0.0.2`](https://github.com/0xPolygon/cdk-validium-node/releases/tag/v0.0.2)

**Comment by Jasun0925** [comment]
@UrsaMajor-t I have given up my resistance and am using [rollup](https://github.com/0xPolygonHermez/zkevm-node) now. This github warehouse is updating slowly and there are still many unresolved issues

**Comment by bap2pecs** [comment]
make sure you set `genesisBlockNumber` correct in genesis.config.json
